include/ngp_algorithms/EnthalpyEffDiffFluxCoeffAlg.h

```diff
@@ -0,0 +1,54 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2014 Sandia Corporation.                                    */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+
+#ifndef EnthalpyEffDiffFluxCoeffAlg_h
+#define EnthalpyEffDiffFluxCoeffAlg_h
+
+#include<Algorithm.h>
```

Comment:
Prefer quotes here... but as long as it is either all quotes or all angle brackets.